test/fixtures.js

```diff
@@ -242,4 +272,10 @@ const notification = {
   subscription_url: 'https://api.github.com/notifications/threads/1234/subscription',
 }
 
-module.exports = { task, pullRequest, issue, notification }
+module.exports = {
+  task: Object.freeze(task),
+  anotherTask: Object.freeze(anotherTask),
+  pullRequest: Object.freeze(pullRequest),
+  issue: Object.freeze(issue),
+  notification: Object.freeze(notification),
+}
```

Comment:
I wanted to try and prevent myself from accidentally changing filters. Freeze isn't enough though, we need something like deepFreeze.